From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Sun, 2 Aug 2026 19:28:41 +0700
Subject: [PATCH] Revert "Add sourceType, remoteId, and remoteServerId to
 recently played entity"

This reverts commit 0cc2d74a24eb3fb4c96cdebe8b0bb5472cec900f.

diff --git a/lib/data/repositories/recently_played_repository.dart b/lib/data/repositories/recently_played_repository.dart
index 76e8363e4..37692717b 100644
--- a/lib/data/repositories/recently_played_repository.dart
+++ b/lib/data/repositories/recently_played_repository.dart
@@ -711,9 +711,6 @@ class RecentlyPlayedRepository {
       album: entity.album,
       filePath: entity.filePath,
       dateAdded: entity.dateAdded,
-      sourceType: entity.sourceType,
-      remoteId: entity.remoteId,
-      remoteServerId: entity.remoteServerId,
     );
   }
 
-- 
2.55.0

